tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -631,6 +631,9 @@ def process_action(self, url, payload):
             raise ValueError("Unknown action %s" % action)
         try:
             result = action_handler(payload)
+        except NotImplementedError:
+            self.logger.warning("Action %s not implemented" % action)
+            self._send_message("complete", "not-implemented")
```

Comment:
I'm not so sure about this. It looks like `not-implemented` will be a new status. Do we need to define it somewhere?

A less risky way is to use the existing `error` status, and put "action %s not implemented" into the message field.

Reply:
> A less risky way is to use the existing error status, and put "action %s not implemented" into the message field.

Done.